src/utils/hwWallet.js

```diff
@@ -16,31 +16,10 @@ const util = require('util');
 const { ipc } = window;
 
 if (ipc) { // On browser-mode is undefined
-  // ipc.on('ledgerConnected', () => {
-  //   store.dispatch(infoToastDisplayed({ label: HW_MSG.LEDGER_CONNECTED }));
-  // });
-
-  // ipc.on('ledgerDisconnected', () => {
-  //   console.log('ledgerDisconnected');
-  //   store.dispatch(errorToastDisplayed({ label: HW_MSG.LEDGER_DISCONNECTED }));
-  // });
-
   ipc.on('ledgerButtonCallback', () => {
     // store.dispatch(infoToastDisplayed({ label: HW_MSG.LEDGER_ASK_FOR_CONFIRMATION }));
   });
 
-  ipc.on('trezorConnected', (event, data) => {
-    store.dispatch(infoToastDisplayed({
-      label: util.format(HW_MSG.TREZOR_CONNECTED, data.model, data.label),
-    }));
-  });
-
-  ipc.on('trezorDisconnected', (event, data) => {
-    store.dispatch(errorToastDisplayed({
-      label: util.format(HW_MSG.TREZOR_DISCONNECTED, data.model, data.label),
-    }));
-  });
-
   ipc.on('trezorButtonCallback', (event, data) => {
     store.dispatch(infoToastDisplayed({
```

Comment:
There are still some more `store.dispatch`s in this util file. IMO it doesn't make sense why are some `ipc.on`s here and some in the middleware and all `ipc.on`s should be together in the middleware.